.doc_gen/metadata/rds_metadata.yaml

```diff
@@ -1,4 +1,20 @@
 # zexi 0.4.0
+rds_Hello:

```

Comment:
There is also an open PR which adds this section for .NET, we may conflict depending on which gets merged first.